core/templates/domain/state/StateObjectFactory.ts

```diff
@@ -166,8 +169,8 @@ export class StateObjectFactory {
   get NEW_STATE_TEMPLATE(): StateBackendDict {
     return constants.NEW_STATE_TEMPLATE as StateBackendDict;
   }
-
-  createDefaultState(newStateName: string): State {
+  // Passes newstatename as null untill new question is created it will be null.
```

Comment:
I don't understand this, how is state connected to a question. Also, please proofread your comments.

Reply:
Done